docs/cvl/methods.md

```diff
@@ -706,6 +709,13 @@ The behavior of the `AUTO` summary depends on the type of call[^opcodes]:
   [State Mutability](https://docs.soliditylang.org/en/v0.8.12/contracts.html#state-mutability)
   in the Solidity manual for details.
 
+(assert-false-summary)=
+#### `ASSERT_FALSE` summaries
+
+This summary essentially just checks that the summarized method is not reached, 
+which can be useful in the presence of unresolved calls that can be dispatched in many ways.
+It can enable more optimizations in the prover and lead to shorter running times.
```

Comment:
```suggestion
This summary checks that the summarized method is not reached, by triggering an assertion failure if a call site to the summarized function is reached.
It can be particularly useful in combination with dispatch list summaries as its default case---to safely rule out calls to methods that do not appear in the dispatch list. In certain cases it can help for optimizations that shorten running times.
```